hotfix/munigeo-requirement

Anthon98 opened the pull request:
# Turku Munigeo Version Requirement Fix

## Added a lock requirement version for django-munigeo-turku above or equal to 0.2.31 for requirements.in file.
-----------------------------------------------------------------------------------------------
### Breakdown:

#### Changes
 1. requirements.in
     * Added django-munigeo-turku and made it accept any version above or equal to 0.2.31
 2. requirements.txt
     * Generated a new requirements.txt file based on the new change.

Commit:
Fixed munigeo requirement version

ezkat commented on requirements.in:
```diff
@@ -15,6 +15,7 @@ django-leaflet
 django-modeltranslation
 django-mptt
 django-munigeo
+django-munigeo-turku>=0.2.31
```
Lock it to 0.2.31 instead.

ezkat commented on requirements.txt:
```diff
@@ -6,83 +6,266 @@
+    #   django-munigeo
+    #   django-munigeo-turku
+    #   django-orghierarchy
+django-munigeo-turku==0.3.6
```
Version 0.3.6 is broken.

ezkat reviewed (changes requested):
django-munigeo-turku 0.3.6 is broken, locking and downgrading it to 0.2.31 is required.

pelmat committed:
Package changes and version fixes

Added missing aiohttp to .in
Removed duplicate django-munigeo
Changed django-munigeo-version to the correct one
Changed django-orghierarchy to it's turkufied version
Updated the version of django for security fixes according to Snyk
Re-compiled requirements.txt correctly with pip-compile

pelmat committed:
Package version changes

Updated lxml due to security issues according to snyk
